pkg/plugins/plugins.go

```diff
@@ -144,12 +148,16 @@ func checkLocalPluginManifest(descriptor LocalDescriptor) error {
 		errs = multierror.Append(errs, fmt.Errorf("%s: unsupported type %q", descriptor.ModuleName, m.Type))
 	}
 
-	if m.Import == "" {
-		errs = multierror.Append(errs, fmt.Errorf("%s: missing import", descriptor.ModuleName))
-	}
+	if m.IsYaegiPlugin() {
+		if m.Import == "" {
+			errs = multierror.Append(errs, fmt.Errorf("%s: missing import", descriptor.ModuleName))
+		}
 
-	if !strings.HasPrefix(m.Import, descriptor.ModuleName) {
-		errs = multierror.Append(errs, fmt.Errorf("the import %q must be related to the module name %q", m.Import, descriptor.ModuleName))
+		if !strings.HasPrefix(m.Import, descriptor.ModuleName) {
+			errs = multierror.Append(errs, fmt.Errorf("the import %q must be related to the module name %q", m.Import, descriptor.ModuleName))
+		}
+	} else if m.WasmPath == "" {
```

Comment:
Looks redudant, if `m.IsYaegiPlugin()` returns false it means that m.WasmPath is not empty.

Reply:
actually, if wasmpath is empty it will always think that the plugin is yaegi plugin. So basically this else is not executed at all. I will remove this

Reply:
Not sure this was resolved